src/tasks/package.js

```diff
@@ -32,41 +43,44 @@ module.exports = (config) => {
   json('package.json')
     .merge({
       main: 'dist/cjs.js',
-      files: [
-        'dist',
-      ],
+      files: ['dist'],
       engines: {
-        // Some versions are skipped because of known issues, see https://github.com/webpack-contrib/organization/issues/7
-        node: `>= ${config.minNode} < 5.0.0 || >= 5.10`,
+        node: `>= ${config.minNode} || >= ${config.latestNodeLTS}`,
       },
       peerDependencies: {
-        webpack: '^2.0.0 || ^3.0.0',
+        webpack: '^3.0.0 || ^4.0.0',
       },
       scripts: {
         start: 'npm run build -- -w',
-        'appveyor:test': 'npm run test',
-        build: "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
+        build:
+          "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js' --copy-files",
         clean: 'del-cli dist',
+        commitlint: 'commitlint',
+        commitmsg: 'commitlint -e $GIT_PARAMS',
         lint: 'eslint --cache src test',
+        'lint:commits':
+          "commitlint --from=$(git merge-base master '${CIRCLE_BRANCH}')",
         'lint-staged': 'lint-staged',
         prebuild: 'npm run clean',
         prepublish: 'npm run build',
         release: 'standard-version',
+        'release:ci': 'conventional-github-releaser -p angular',
+        'rrelease:validate':
```

Comment:
`rrelease` => `release` (typo)

Reply:
It's fixed upstream as well as in `next`